man/coap_resource.txt.in

```diff
@@ -292,6 +297,10 @@ the _resource_ definion.
 The *coap_get_resource_from_uri_path*() function is used to return the resource
 identified by the unique string _uri_path_ associated with _context_.
 
+*Function: coap_print_wellknown()*
+
+The *coap_print_wellknown*() function prints the names of all known resources
+of the given _context_ into _buf_.
```

Comment:
of the given _context_ into _buf_ which has a maximum size of _buflen_.  The first _offset_ bytes are skipped from the output to handle block transfers. The _query_filter_ is usually defined by the CoAP Uri-Query options as a query.